src/mlpack/methods/ann/init_rules/glorot_init.hpp

```diff
@@ -67,8 +67,9 @@ class GlorotInitializationType
    * @param rows Number of rows.
    * @param cols Number of columns.
    */
-  template<typename eT>
-  void Initialize(arma::Mat<eT>& W,
+  template<typename MatType,
+      typename std::enable_if_t<IsMat<MatType>::value, bool> = false>
```

Comment:
The `enable_if` isn't needed here, since this already will work only for matrices due to the two `size_t` arguments (as compared with the three arguments for the `Cube` version).